MobileCenterDistribute/MobileCenterDistributeTests/MSDistributeUtilTests.m

```diff
@@ -43,4 +48,268 @@ - (void)testLocalizedString {
   XCTAssertTrue([test isEqualToString:@"Ignore"]);
 }
 
+- (void)testCompareReleaseSameVersion {
+  
+  /**
+   * If.
+   */
+  
+  // Mock current UUID.
+  NSString *testUUID = @"CD55E7A9-7AD1-4CA6-B722-3D133F487DA9";
+  id parserMock = OCMClassMock([MSBasicMachOParser class]);
+  OCMStub([parserMock machOParserForMainBundle]).andReturn(parserMock);
+  OCMStub([parserMock uuid]).andReturn([[NSUUID alloc] initWithUUIDString:testUUID]);
+  
+  // Mock current versions.
+  NSString *expectedShortVer = @"2.5.3-alpha+EF69A";
+  NSString *expectedVersion = @"2.5.3.1";
+  NSDictionary<NSString *, id> *plist = @{
+                                          @"CFBundleShortVersionString" : expectedShortVer,
+                                          @"CFBundleVersion" : expectedVersion
+                                          };
+  id bundleMock = OCMClassMock([NSBundle class]);
+  OCMStub([bundleMock mainBundle]).andReturn(bundleMock);
+  OCMStub([bundleMock infoDictionary]).andReturn(plist);
+  
+  // Define release details.
+  MSReleaseDetails *details = [MSReleaseDetails new];
+  details.shortVersion = expectedShortVer;
+  details.version = expectedVersion;
+  details.packageHashes = @[ [testUUID lowercaseString] ];
```

Comment:
This is not a right format anymore. Can you add FIXME comment for all places that are using UUID format? Or, you can change it to something else.